canonical-websites/www.ubuntu.com
# Flatten legal service-terms section

## Done

- Flattened legal service-terms section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/ubuntu-advantage-service-terms](http://0.0.0.0:8001/legal/ubuntu-advantage-service-terms) and [the old link](http://0.0.0.0:8001/legal/service--terms) redirects
* test the link [/legal/ubuntu-advantage-service-terms](http://0.0.0.0:8001/legal/ubuntu-advantage-service-terms) and [the old link](http://0.0.0.0:8001/legal/service--terms/ja) redirects
* see that I hid the ja and older versions in the navigation.yaml

## Issue / Card

Fixes #4196
